Ato-visionOS/Ato-visionOS/Source/View/SplitPeriodicView.swift

```diff
@@ -15,58 +15,101 @@ import SwiftUI
 struct SplitPeriodicView: View {
     
     // MARK: - State
-    /// 사용자가 선택한 원소
-    @State private var selectedElement: Element? = nil
+    // 사용자가 선택한 원소
+    @State private var selectedAtom: DetailAtomModel? = nil
 
     // MARK: - Body
     var body: some View {
-        GeometryReader { geometry in
+        GeometryReader {geometry in
+            let size = geometry.size
+            let width: CGFloat = geometry.size.width
+            let height: CGFloat = geometry.size.height
+//            let dynamicCorner = min(30, min(width, height) * 0.05)
             
-            // MARK: - 레이아웃 계산
-            
-            let totalWidth = geometry.size.width
-            let leftWidth = totalWidth * 0.7           // 좌측 주기율표 영역
-            let rightWidth = totalWidth - leftWidth    // 우측 상세보기 영역
-            
-            HStack(spacing: 0) {
-                
+            HStack(spacing: 10) {
                 // MARK: - 주기율표 뷰 (왼쪽)
+                
                 PeriodicTableView(
-                    selectedElement: $selectedElement,
+                    selectedAtom: $selectedAtom,
                     elementsGrid: elementsGrid,
-                    containerWidth: leftWidth,
-                    containerHeight: geometry.size.height
+                    width: width * 0.71,
+                    height: height
                 )
-                .frame(minWidth: leftWidth)
-
-                Divider() // 가운데 구분선
+                .frame(width: width * 0.71, height: height)
+                .background(
+                  LinearGradient(
+                    stops: [
+                      Gradient.Stop(color: .white.opacity(0.37), location: 0.00),
+                      Gradient.Stop(color: Color(red: 0.45, green: 0.45, blue: 0.45).opacity(0.42), location: 0.95),
+                    ],
+                    startPoint: UnitPoint(x: 0, y: -0.06),
+                    endPoint: UnitPoint(x: 1, y: 1)
+                  )
+                )
```

Comment:
```suggestion
                .bg()
```
하시면 자동으로 뒷배경 들어가게 모디파이어 만들어두었씁니다! 첫번째 pr에서 제가 설명 적어뒀었어요!